integration-tests/features/src/utils.py

```diff
@@ -14,3 +15,52 @@ def download_file_from_url(url):
 def split_comma_separated_list(l):
     """Split the list into elements separated by commas."""
     return [i.strip() for i in l.split(',')]
+
+
+def oc_login(url, username, password, tls_verify=True):
+    """Wrapper around `oc login`.
+
+    :param url: str, OpenShift URL
+    :param username: str, username
+    :param password: str, password
+    :param tls_verify: bool, verify server's certificate?; default: True
+    :return: None on success, raises `subprocess.CalledProcessError` on error
+    """
+    command = ['oc', 'login', url, '--username', username, '--password', password]
+    if not tls_verify:
+        command.extend(['--insecure-skip-tls-verify=true'])
+
+    try:
+        subprocess.check_call(command)
+    except subprocess.CalledProcessError as e:
+        # replace password with '***' so somebody will not accidentally leak it in CI logs
+        e.cmd = [x if x != password else '***' for x in e.cmd]
```

Comment:
It won't work if the password is part of longer string, therefore:

```python
e.cmd = [x.replace(password, '***') for x in e.cmd]
```

Reply:
Your version is simpler and easier to read, updated. Thanks :+1: